.github/workflows/nightly-release.yml

```diff
@@ -0,0 +1,106 @@
+# **what?**
+# Nightly releases to GitHub and PyPI. This workflow produce the following outcome:
```

Comment:
```suggestion
# Nightly releases to GitHub and PyPI. This workflow produces the following outcome:
```

Reply:
Fixed in https://github.com/dbt-labs/dbt-core/pull/6602/commits/2fbde2cc196d0f942da42cb03b54b25be4e95e53